target/scripts/update-check.sh

```diff
@@ -41,20 +38,21 @@ There is a docker-mailserver update available on your host: $(hostname -f)
 Current version: ${VERSION}
 Latest  version: ${LATEST}
 
-Changelog: ${CHANGELOG}
+Changelog: ${CHANGELOG_URL}
 EOM
       echo "${MAIL}" | mail -s "Mailserver update available! [ ${VERSION} --> ${LATEST} ]" "${POSTMASTER_ADDRESS}" && \
 
-      _log "Info: Update available [ ${VERSION} --> ${LATEST} ]" && \
+      _log 'info' "Update available [ ${VERSION} --> ${LATEST} ]"
```

Comment:
The removed `&& \` was on purpose, to only exit when notification was successful.